documentation/github-project-guidelines.md

```diff
@@ -0,0 +1,48 @@
+﻿# Guidelines for our GitHub projects
+
+In this place we will have the documentation for how to handle issues
+among projects of **TecnoYucas**.
+
+## General notes
+
+### Some rules first about the project
+
+1. Every repo must have a automatic Kanban-like GitHub project
+2. Every single pull request MUST be associated with a GitHub issue
+3. Whenever you close a pull request or comment on an issue, please
+make sure to agree on issue completion.
+4. All issues should have a deliverable product (a document or a
+merged branch, this would help on issue completion agreement).
+
+### Some recommendations
+
+- Keep language as complete and concise as possible.
+- Use Todo lists (this can help to better define the scope of an issue
+  and the state of it).
+- Please try not to create huge issues, make them as short and clear
+  scoped as possible. If you need some sort of complex issue, please
```

Comment:
I think `scoped` slipped in here :sweat_smile:

Reply:
@aitbw Why do you say it is slipped?

Reply:
Never heard of `short and clear scoped` before